### ISSUE TITLE
[BUG]branch-0.6-release: In pca_demo, singular_values are NOT equal

### ISSUE THREAD

**Issue by taureandyernv** [rapidsai/notebooks#84]
In code block 10, when comparing cuml to scikitlearn, comparing singular values returns "not equal"

```
compare pca: cuml vs sklearn          singular_values_ NOT equal
compare pca: cuml vs sklearn               components_ equal
compare pca: cuml vs sklearn       explained_variance_ equal
compare pca: cuml vs sklearn explained_variance_ratio_ equal
```

Ran twice to confirm.

__ONLY__ happens when using the mortgage dataset.   the random values case returns `equal`

**Comment by taureandyernv** [comment]
Onur wanted me to add that these were working from a docker container built from branch-0.6 source on 03/11.

**Comment by taureandyernv** [comment]
closed as this was fixed in 0.6
